src/cache_cf.cc

```diff
@@ -418,6 +422,12 @@ EvalBoolExpr(const char* expr)
     return false; // this place cannot be reached
 }
 
+static SBuf
+ConfigPosition()
```

Comment:
Please convert into an ostream manipulator to reduce configuration-time dependency on complex (and debugging-rich) things like SBuf:

```suggestion
std::ostream &ConfigPosition(std::ostream &os)
```